src/Parser/DeathKnight/Frost/Modules/Items/Tier21_2p.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import getDamageBonus from 'Parser/Mage/Shared/Modules/GetDamageBonus';
+
+/**
+ * Howling Blast damage increased by 15% and Obliterate damage increased by 15% and Frostscythe damage increased by 15%
+ */
+
+const DAMAGE_BONUS = 0.15;
+
+class Tier21_2p extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  obliterateDamage = 0;
+  howlingblastDamage = 0;
+  frostscytheDamage = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.FROST_DEATH_KNIGHT_T21_2SET_BONUS.id);
+  }
+
+  on_byPlayer_damage(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.OBLITERATE_MAIN_HAND_DAMAGE.id || spellId === SPELLS.OBLITERATE_OFF_HAND_DAMAGE){
+      this.obliterateDamage += getDamageBonus(event, DAMAGE_BONUS);
+      return;
+    }
+    if(spellId === SPELLS.HOWLING_BLAST.id){
+      this.howlingblastDamage += getDamageBonus(event, DAMAGE_BONUS);
+      return;
+    }
+    if(spellId === SPELLS.FROSTSCYTHE_TALENT.id){
```

Comment:
It's fairly common for talent spells to have the talent ID be the cast ID but *not* the damage ID, in which case this would be the wrong ID. I'm going to assume that you've checked and they are the same, but if you haven't checked I recommend doing so.